_pages/business-units/18f/projects-partners/collaboration-tools.md

```diff
@@ -0,0 +1,168 @@
+
+**[Trello]({{site.baseurl}}/trello/)** is a project management tool that organizes information into lists. Teams often use it to track specific tasks through different stages (such as backlog, in progress, in review, and done). Trello boards can be private or public.
+
+<details>
+<summary>Compliance and records considerations:</summary>
+* GSA Tech Portfolio is working on a FedRAMP application as of January 2020. Members of TTS can use it and will be notified if anything changes.
+* Trello boards should be considered records.
+</details>
+
+<details>
+<summary>Trello alternatives</summary>
+* **Zenhub:** Not recommended from a compliance perspective. No FedRAMP in progress, no GSA IT approval for use of the service. From a user perspective, it’s great that it works within GitHub and provides higher-level scoping and linking (e.g., epics and dependencies) than Jira does.
+* **GitHub Projects:** Not recommended from usability/UI perspective, as the interface is basic without the richness of other planning systems or the simplicity of Trello. Compliance/records considerations are the same as for GitHub.
+* **Jira:** The GSA IT Jira instance is not recommended. It’s pretty difficult to grant access to outsiders, and it’s pretty heavyweight and difficult to configure.
+* **Microsoft Azure Devops Boards:** Bundled with Azure, but not within FedRAMP services in scope.
+</details>
+
+* **IF** not part of partners’ daily workflow/practices, **THEN** discuss task progress in sprint ceremonies or via email.
+* **IF** blocked by firewall/security issues, **THEN** build a board using whiteboarding or visualization tools like Mural, Google Slides or Powerpoint. 
+
+### Viewing/scheduling calendars
+
+**[Google Calendar]({{site.baseurl}}/google-calendar/)** is the calendar tool that comes with gmail and G Suite. The Handbook has guidelines and requirements for providing partners with access. 
+
+<details>
+<summary>Compliance and records considerations</summary>
+* [Google G Suite has a FedRAMP Agency ATO](https://marketplace.fedramp.gov/#/product/google-g-suite) Moderate Impact authorized since January 2016. Agencies using the service include GSA, DOE, DHS, and others.
+</details>
+
+Here are some common barriers to using Google Calendar and some workarounds to try:
+* **IF** G-suite is incompatible with partners’ available tools, **THEN** have partners schedule meetings, try Max.gov calendar, or use email, phone, or Slack to align on meeting times.
+
+### Development and Documentation
+
+**[GitHub]({{site.baseurl}}/github/)** is an online code storage and collaboration platform. 18F will create a repository (commonly called a repo) for the project and use it to store both documents and code. By default, 18F’s work with your team on GitHub will be visible to the public. Your 18F team can tell you more about the benefits of working in the open. Teams may also use GitHub, as well as browser add-ons that complement it, to collaboratively track project tasks.
```

Comment:
I'm wondering if we should note that we may need to work within the partner's existing SVN system, or that we may need to think about transferring ownership of the repo to them. It's probably OK for now and I'm just thinking out loud.